src/components/calcite-input-time-picker/calcite-input-time-picker.e2e.ts

```diff
@@ -0,0 +1,153 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { accessible, defaults, focusable, reflects, renders } from "../../tests/commonTests";
+import { formatNumberAsTimeString } from "../calcite-time-picker/utils";
+
+describe("calcite-input-time-picker", () => {
+  it("renders", async () => renders("calcite-input-time-picker"));
+
+  it("is accessible", async () => accessible(`<calcite-input-time-picker></calcite-input-time-picker>`));
+
+  it("has defaults", async () =>
+    defaults("calcite-input-time-picker", [
+      { propertyName: "active", defaultValue: false },
+      { propertyName: "disabled", defaultValue: false },
+      { propertyName: "scale", defaultValue: "m" },
+      { propertyName: "step", defaultValue: 60 },
+      { propertyName: "theme", defaultValue: "light" }
```

Comment:
Lets not set a default theme. Maybe we should update conventions to include not setting it by default?